app/styles/.scss-lint.yml

```diff
@@ -17,6 +17,9 @@ linters:
     enabled: true
     style: include_zero # or 'exclude_zero'
 
+  PlaceholderInExtend:
+    enabled: false
+
```

Comment:
This gets rid of the `[W] Prefer using placeholder selectors (e.g. %some-placeholder) with @extend` warnings from [**scss-lint**](https://github.com/causes/scss-lint).